functions

```diff
@@ -254,7 +256,7 @@ index_of() {
     local item=$1; shift
 
     for (( _idx=1; _idx <= $#; _idx++ )); do
-        if [[ $item = ${!_idx} ]]; then
+        if [[ $item == ${!_idx} ]]; then
```

Comment:
The `==` isn't needed ... but the quotes were not added

Reply:
I guess I was too afraid of breaking things.